aspnetcore/tutorials/ai-powered-group-chat/ai-powered-group-chat.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build an AI-Powered Group Chat with SignalR and OpenAI
+author: kevinguo-ed
+description: A tutorial explaining how SignalR and OpenAI are used together to build an AI-powered group chat
+ms.author: kevinguo
+ms.date: 08/27/2024
+uid: tutorials/ai-powered-group-chat
+---
+
+## Overview
+
+The integration of AI into applications is rapidly becoming a must-have for developers looking to help their users be more creative, productive and achieve their health goals. AI-powered features, such as intelligent chatbots, personalized recommendations, and contextual responses, add significant value to modern apps. The AI-powered apps that came out since ChatGPT captured our imagination are primarily between one user and one AI assistant. As developers get more comfortable with the capabilities of AI, they are exploring AI-powered apps in a team's context. They ask "what value can AI add to a team of collaborators"? 
+
+This tutorial guides you through building a real-time group chat application. Among a group of human collaborators in a chat, there's an AI assistant which has access to the chat history and can be invited to help out by any collaborator when they start the message with `@gpt`. The finished app looks like this. 
+
+:::image type="content" source="./ai-powered-group-chat.jpg" alt-text="user interface for the AI-powered group chat":::
+
+We use OpenAI for generating intelligent, context-aware responses and SignalR for delivering the response to users in a group. You can find the complete code [in this repo](https://github.com/microsoft/SignalR-Samples-AI/tree/main/AIStreaming).
+
+## Dependencies
+You can use either Azure OpenAI or OpenAI for this project. Make sure to update the `endpoint` and `key` in `appsetting.json`. `OpenAIExtensions` reads the configuration when the app starts and they are required to authenticate and use either service.
```

Comment:
This needs to be

Update the `endpoint` and `key` in `appsetting.json` file as shown in the following code:

After this PR merges, we can move the code to a sample folder and display the `appsetting.json` with the changes highlighted.